src/Giraffe/HttpHandlers.fs

```diff
@@ -377,6 +377,35 @@ let negotiateWith (negotiationRules    : IDictionary<string, obj -> HttpHandler>
                         |> fun mimeType -> negotiationRules.[mimeType.MediaType.Value]
                         |> fun handler  -> handler responseObj next ctx
 
+
+
+let DefaultNegotiationRules = 
```

Comment:
why are we creating a list, than adding to dictionary? would the prior syntax of `dict [ ... ]` not be better as module values (this list) will be persisted when not needed after dictionary construction? not too big a deal and might be to allow further additions  (as dict is read-only)? in this case we should probably define `DefaultNegotiationRules` as scoped variable in `NegotiationRules` and additionally, we should pass the count to the Dictionary constructor to avoid re-sizing of known dataset?